strings.js

```diff
@@ -0,0 +1,5 @@
+var firstName = "Peter";
+var lastName = "Parker";
+var fullName = firstName + " " + lastName;
```

Comment:
hey! i would recommend that you using the latest JSX version with string templates, just like the example below!
const fullName = `${firstName} ${lastName}`